packages/vue3/src/components/Divider/Divider.vue

```diff
@@ -0,0 +1,44 @@
+<script setup lang="ts">
+import type { Orientation, Style } from './types';
+
+withDefaults(
+  defineProps<{
+    orientation?: Orientation
+    borderStyle?: Style
+  }>(),
+  {
+    orientation: 'horizontal',
+    borderStyle: 'solid',
+  },
+);
+</script>
+
+<template>
+  <div :class="{ [orientation]: true, [borderStyle]: true }" />
```

Comment:
In this use case, we can utilize the array syntax
```suggestion
  <div :class="[orientation, borderStyle]" />
```